src/rust/engine/process_execution/src/docker_tests.rs

```diff
@@ -748,6 +748,7 @@ async fn run_command_via_docker_in_dir(
   let stderr_bytes = store
     .load_file_bytes_with(original.stderr_digest, |bytes| bytes.to_vec())
     .await?;
+  runner.shutdown().await?;
```

Comment:
Correct which is why the PR description states:
> For Docker tests, explicitly call the shutdown method. Docker containers may remain due to test failure. This is an intentional choice to avoid having to solve async drops (and blocking in place) in the test context.

Reply:
Yea. I guess I'm just pointing out that you _can_ still tear down in case of errors (but not cancellation) if you want.

Reply:
Good point.

Reply:
Done.